README.md

```diff
@@ -817,6 +817,33 @@ _You can enable the following settings in Xcode by running [this script](resourc
   }
   ```
 
+* <a id='wrap-guard-else'></a>(<a href='#wrap-guard-else'>link</a>) **Break before the `else` keyword in a multi-line guard statement.** [![SwiftFormat: elseOnSameLine](https://img.shields.io/badge/SwiftFormat-elseOnSameLine-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#elseOnSameLine )
+
+  <details>
+
+  ```swift
+  // WRONG -- else should be on its own line for multi-line guard statements
```

Comment:
In https://github.com/airbnb/swift/pull/227 we use parens for explaining why this is wrong, should we use that here too?
```suggestion
  // WRONG (else should be on its own line for multi-line guard statements)
```

Reply:
Looks like in other example like this we also use parenthesis, so I'll update this to do that instead